src/com/scottyplunkett/server/Cycle/RequestResponseCycle.java

```diff
@@ -1,33 +1,33 @@
 package com.scottyplunkett.server.Cycle;
 
 import com.scottyplunkett.server.Cycle.Request.HTTPRequest;
+import com.scottyplunkett.server.Cycle.Response.Behavior.Behavior;
```

Comment:
Behavior.Behavior, eh? 🤔

Reply:
> Package names are written in all lower case to avoid conflict with the names of classes or interfaces. Companies use their reversed Internet domain name to begin their package names—for example, com.example.mypackage for a package named mypackage created by a programmer at example.com .

WHOOPS! I should change ALL my package names to lowercase, I'll put in a separate PR just for that. Good catch, thanks.